tests/ast/test_ast_driver.py

```diff
@@ -107,6 +107,12 @@ def indent_lines(source: str, n_indents: int = 0):
     return "\n".join(map(lambda line: indent * n_indents + line, source.split("\n")))
 
 
+def normalize_temp_names(s: str):
+    return re.sub(
```

Comment:
can we add a docstring to describe what this function does? `normalize` itself is not very descriptive...